### PR TITLE
Add unit tests for GIBCT containers

### PR DESCRIPTION
**Pull request by bshyong**
resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2466

### CODE REVIEW

File: test/gibct/containers/LandingPage.unit.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { createStore } from 'redux';
+import { expect } from 'chai';
+import React from 'react';
+import sinon from 'sinon';
+import SkinDeep from 'skin-deep';
+
+import { LandingPage } from '../../../src/js/gi/containers/LandingPage';
+import reducer from '../../../src/js/gi/reducers';
+
+const store = createStore(reducer);
```
Comment (U-DON) [inline test/gibct/containers/LandingPage.unit.spec.js]: In the test for `GiBillApp`, you set `defaultProps` in one line instead referring to it later as `store.getState()`. Probably should be consistent here (I prefer the `defaultProps`).

File: test/gibct/containers/Modals.unit.spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import SkinDeep from 'skin-deep';
+
+import Modals from '../../../src/js/gi/containers/Modals';
+import reducer from '../../../src/js/gi/reducers';
+
+const store = createStore(reducer);
+
+describe('<Modals>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(<Modals store={store}/>);
```
Comment (U-DON) [inline test/gibct/containers/Modals.unit.spec.js]: Why doesn't this require `getState()` like the others?

**Review by U-DON — APPROVED** [review]
LGTM. Couple of minor comments.

File: test/gibct/containers/LandingPage.unit.spec.js (hunk 1 above)
Comment (bshyong) [inline test/gibct/containers/LandingPage.unit.spec.js]: Will do

File: test/gibct/containers/Modals.unit.spec.js (hunk 2 above)
Comment (bshyong) [inline test/gibct/containers/Modals.unit.spec.js]: This container is only bringing in Modal components which are already covered by common component tests. So here we are testing the Modals connected component directly (passing in store) and doing a simple render check.

In the other container tests, I am testing the undecorated component, so we need to manually pass in the props from the reducers. That said, I can refactor this test to follow the same pattern as the others for consistency.

Comment (U-DON) [inline test/gibct/containers/Modals.unit.spec.js]: Oh, didn't notice till now that this is the decorated component. Sure, would like to see it tested as a regular component like the others.

**Pull request merged**